paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormViewModel.kt

```diff
@@ -132,14 +131,6 @@ internal class FormViewModel @Inject internal constructor(
         externalHiddenIdentifiers.value = identifierSpecs
     }
 
-    private val sectionToFieldIdentifierMap = layout.items
```

Comment:
Now that save for future use is removed, only the address hides fields inside a section